doc/usecases/mount/UC_mount.md

```diff
@@ -0,0 +1,43 @@
+# Use Case: Mount a data source
+
+## Summary
+
+- **Scope:** Mounting
+- **Level:** Kite
+- **Actors:** Administrators
+- **Brief:** The administrators can include configuration data from different data sources into Elektra.
+- **Status:** Draft (currently only files are supported)
+
+## Scenarios
+
+- **Precondition:**
+
+  - The administrators have a working installation of Elektra and a tool to access the KDB (key database) on their system.
+
+- **Main success scenario:**
+
+  - The administrators specify a mountpoint in the KDB and an identifier for the data source they want to mount.
+    - For file-based data sources, the identifier is usually the path and filename.
+  - The data source is successfully mounted into the KDB.
+
+- **Alternative scenario:**
+
+  - The given data source is not found. (e.g. wrong filename)
+    - Elektra reports the error to the administrators.
```

Comment:
There is no need that a mounted config file needs to be present. What is an alternative scenario, though is:
```suggestion
  - There are already keys present at the requested mountpoint.
    - Elektra reports the error to the administrators, unless `--strategy` is used, then keys are merged as requested.
```

Reply:
I'm not sure if this what you meant, but your suggestion sounds like: We tried e.g. `kdb mount foo user:/foo`, but keys below `user:/foo` already existed (without a mountpoint at `user:/foo`). If you meant, there was already a mountpoint at `user:/foo`, I'd put it like this for more clarity:

```suggestion
  - The desired mountpoint already exists with a different configuration.
    - Elektra reports the error to the administrators, unless `--strategy` is used, then keys are merged as requested.
```